src/bentoml/_internal/log.py

```diff
@@ -87,6 +87,7 @@ def configure_logging():
         CLI_LOGGING_CONFIG["loggers"]["bentoml"]["level"] = logging.ERROR
         CLI_LOGGING_CONFIG["root"]["level"] = logging.ERROR
     elif get_debug_mode():
+        CLI_LOGGING_CONFIG["handlers"]["defaulthandler"]["level"] = logging.DEBUG
```

Comment:
What does this fix?

I tried `--debug` and non and see no difference?

Reply:
The debug logs doesn't print with `--debug` flag on the CLI context, you can see it with `bentoml build`